manifests/init.pp

```diff
@@ -93,22 +96,26 @@
   # Same package name for both redhat based and debian based
   package{ $pgbouncer_package_name:
     ensure  => installed,
-    require => [ Class['postgresql::repo::yum_postgresql_org'], Anchor['pgbouncer::begin'] ],
+    require => [
+      Class[
+        'postgresql::repo::yum_postgresql_org'],
+        Anchor['pgbouncer::begin']
+      ],
   }
 
   # verify we have config file managed by concat
-  concat { "$conffile":
+  concat { $::pgbouncer::conffile:
```

Comment:
You don't need to use the pgbouncer namespace here.  You can simply use `$conffile` without quotes.  Same with the other changes made like this one.